src/js/stores/__tests__/ChronosStore-test.js

```diff
@@ -17,16 +17,21 @@ const JobTree = require('../../structs/JobTree');
 describe('ChronosStore', function () {
 
   beforeEach(function () {
+    // Clean up application timers
+    jasmine.clock().uninstall();
```

Comment:
If the uninstall is called in the `afterEach` do we really need it again in the `beforeEach`? If that is not the case I would like to have it rather in the `afterEach` then in the `beforeEach` since it is a tear down task.